fvtest/porttest/omrstrTest.cpp

```diff
@@ -1479,39 +1478,47 @@ TEST(PortStrTest, str_WinacpToMutf8)
  *         TEST_FAIL on error
  */
 static int
-printToBuffer(struct OMRPortLibrary *portLibrary, BOOLEAN precision, char *buffer, size_t bufferLength, const char *expectedOutput, const char *format, ...)
+printToBuffer(struct OMRPortLibrary *portLibrary, BOOLEAN precision, char *buffer, size_t bufferLength, const char *expectedOutput, int expectedCount, const char *format, ...)
```

Comment:
The parameter `expectedCount` should be added to the function description above.